x-pack/test/api_integration/apis/fleet/delete_agent.ts

```diff
@@ -65,7 +65,7 @@ export default function({ getService }: FtrProviderContext) {
         .delete(`/api/ingest_manager/fleet/agents/agent1`)
         .auth(users.fleet_user.username, users.fleet_user.password)
         .set('kbn-xsrf', 'xx')
-        .expect(403);
+        .expect(404);
```

Comment:
@legrego I believe this change is from adding the tags on the routes. I recall you mentioning something about tags security using 404 vs 403. Is that correct? If so, can you link to any docs about it? Thanks.

Reply:
Yep, This is an expected change based in your adjustments in this PR. We don't have any formal docs around this behavior though